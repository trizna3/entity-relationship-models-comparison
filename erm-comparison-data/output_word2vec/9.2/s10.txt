tile type-EMPTY
tile-tile
move-movement
games;boards-system;game;board
units-units
weather-weather
fuel type-fuel type
CHANGE_CARDINALITY:S-game-system
CONTRACT_11_ASSOCIATION:S-system-game
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-fuel type-movement
CHANGE_CARDINALITY:S-board-game
CONTRACT_11_ASSOCIATION:S-game-board
CONTRACT_11_ASSOCIATION:E-games-boards
REBIND_NARY_ASSOCIATION:E-weather-fuel type-tile type
