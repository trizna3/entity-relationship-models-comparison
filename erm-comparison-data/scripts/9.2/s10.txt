entity set system {
};

entity set game {
};

entity set board {
	"rows"
	"columns"
};

entity set tile {
	"type"
};

entity set units {
	"name"
	"health"
	"attack"
	"defense"
	"points count"
	"fuel"
};

entity set movement {
	"tile type"
	"fuel type"
	"weather type"
	"points needed"
};

entity set weather {
	"move count"
	"type"
};

association between 
{
	system ONE
	game MANY
}{};

association between 
{
	game ONE
	board MANY
}{};

association between 
{
	game ONE
	tile MANY
}{};

association between 
{
	tile ONE
	units MANY
}{};

association between 
{
	weather ONE
	movement MANY
}{};