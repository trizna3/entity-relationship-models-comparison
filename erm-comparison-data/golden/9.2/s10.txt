games-game
boards-board
tile-tile
tile type-
weather-weather
fuel type-fuel
units-units
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-fuel-units